src/Template/Github/create_issue.ctp

```diff
@@ -17,17 +17,9 @@
     <fieldset>
         <legend>Github Issue</legend>
         <?php
+            $milestone_default_val = $pma_version;
```

Comment:
This code probably doesn't make sense with GitHub - we're using milestone for release where the issue has been fixed. But this is really independent to the tests you're adding here.